src/wp-includes/option.php

```diff
@@ -323,36 +323,20 @@ function wp_load_alloptions( $force_cache = false ) {
  * Loads and caches certain often requested site options if is_multisite() and a persistent cache is not being used.
  *
  * @since 3.0.0
- *
- * @global wpdb $wpdb WordPress database abstraction object.
+ * @since 6.0.0 Uses update_meta_cache
  *
  * @param int $network_id Optional site ID for which to query the options. Defaults to the current site.
  */
 function wp_load_core_site_options( $network_id = null ) {
-	global $wpdb;
-
-	if ( ! is_multisite() || wp_using_ext_object_cache() || wp_installing() ) {
```

Comment:
Why wp_using_ext_object_cache() is removed?

Reply:
I think that it can safely be removed, because this check would now be handled by the meta-data API instead?

Reply:
The meta api, checks to see if meta is in the object cache and if not then it loads it from the database. One of the many reason I want to make this change, so much of this stuff is handle upstream.

Reply:
For reference https://github.com/WordPress/wordpress-develop/blob/04e9728701b3dde0260c7cdebfd726edbe62ac97/src/wp-includes/meta.php#L1151-L1191